src/main/resources/db/changelog/20201103112757-purge-current-sca-certs.xml

```diff
@@ -20,8 +20,9 @@
             <column name="cont_acc_cert_id"/> <!-- this sets the column value to null -->
         </update>
 
-        <delete tableName="cp_cont_access_cert" />
-
-        <delete tableName="cp_owner_env_content_access" />
+        <sql>
+            TRUNCATE TABLE cp_cont_access_cert cascade;
```

Comment:
Actually, the cascade here is dangerous. This will truncate any table with a foreign key relationship, which is not what we want to do here -- we could end up clearing half the DB.

Reply:
Also, it doesn't look like mysql or mariadb support cascade on TRUNCATE:

- https://dev.mysql.com/doc/refman/8.0/en/truncate-table.html
- https://mariadb.com/kb/en/truncate-table/